server.go

```diff
@@ -0,0 +1,79 @@
+package sqrl
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"sync"
+)
+
+var mutex = &sync.Mutex{}
+
+func com(w http.ResponseWriter, req *http.Request) {
+	mutex.Lock()
+	c, err := GetCPUInfo()
+	if err != nil {
+		log.Fatal(err)
+	}
+	mutex.Unlock()
+
+	c.LogWriter()
+
+	fmt.Println(c)
+}
+
+func ros(w http.ResponseWriter, req *http.Request) {
+	mutex.Lock()
+	r, o, s, err := RamOSInfo()
```

Comment:
Would you mind breaking RamOSInfo() up?  I would prefer to have one that returned ram, another that did os, and another one that did swap... It shouldn't be too much work.  Name them the same way we have done the others.